lib/ex_crypto.ex

```diff
@@ -188,11 +188,11 @@ defmodule ExCrypto do
   @spec generate_aes_key(atom, atom) :: {:ok, binary} | {:error, binary}
   def generate_aes_key(key_type, key_format) do
     case {key_type, key_format} do
-      {:aes_128, :base64} -> rand_bytes!(16) |> url_encode64
+      {:aes_128, :base64} -> rand_bytes!(16) |> encode64
       {:aes_128, :bytes} -> rand_bytes(16)
-      {:aes_192, :base64} -> rand_bytes!(24) |> url_encode64
+      {:aes_192, :base64} -> rand_bytes!(24) |> encode64
       {:aes_192, :bytes} -> rand_bytes(24)
-      {:aes_256, :base64} -> rand_bytes!(32) |> url_encode64
+      {:aes_256, :base64} -> rand_bytes!(32) |> encode64
```

Comment:
How would you feel about adding `:url_base64` as a `key_format` option so that both `:url_base64` and `:base64` are available?

Reply:
Seems reasonable to me; I'll update to support that. :+1:

Reply:
Circling back; this is still on my todo list, but I've been swamped the past couple of weeks (especially with the schools closed and the family stuck in the house these days.) Will try to get to it this week.

Reply:
OK, done. Sorry it took me so long to get around to it!